pyproject.toml

```diff
@@ -21,13 +21,12 @@ repository="https://github.com/predictive-analytics-lab/pal-conduit"
 readme="README.md"
 
 [tool.poetry.dependencies]
-python = "^3.7.1"
-numpy = "^1.21"
+python = "^3.8.0"
 albumentations = "^1.0.0"
 lightning-bolts = "^0.3.4"
-torch = "^1.9.1"
-torchvision = "^0.10.1"
-torchaudio = "^0.9.1"
+torch = ">=1.9.1"
```

Comment:
why is there the 1.9 dependency? I can't remember

Reply:
Nor can I